AsyncDisplayKit/ASEditableTextNode.h

```diff
@@ -18,6 +19,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface ASEditableTextNode : ASDisplayNode
 
+/**
+ * @abstract Initializes a editable text node with a provided TextKit stack.
```

Comment:
My main concern with this change is that it makes it quite unclear that it is possible to create this object with a standard alloc / init.

Because this method is for a very specialized use case, I would suggest putting it at the very end of the file even though it is against typical convention to put initializers there.  I'm not sure if there is a way to re-declare -init (I tried this recently in a similar case and got a compiler error, but did zero investigation on exactly why)

Let me know if you have any thoughts or opinions on the best way to handle this, but it is definitely a priority for me to make sure the header files remain approachable and intuitive for simple use cases while providing advanced extensibility when desired.